File: 01_Programming_Basics/01_Programming_Basics_with_C++/08_While_Loop_Lab/01_Number_In_Range.cpp
#include <iostream>
using namespace std;

int main() {
  int n = 0;
  
  while (true) {
    cin >> n;

    if (n > 0 && n <= 100) {
      break;
    } else {
      cout << "Invalid Number!" << endl;
    }
  }

  cout << "The number is: " << n << endl;

  return 0;
}

File: 01_Programming_Basics/03_Programming_Basics_with_JavaScript/05_While_Loop/01_NumberInRange.js
function NumberInRange(args) {
  let i = 0;
  let n = parseFloat(args[i]);

  while(true) {

    if (n > 0 && n <= 100) { 
      console.log("The number is: " + n)
      break; 
    }
    
    console.log("Invalid Number");
    i++;
    n = parseFloat(args[i]);
 
  }
}